marklogic-data-hub/src/main/java/com/marklogic/hub/impl/HubProjectImpl.java

```diff
@@ -371,6 +371,7 @@ public Path getCustomMappingFunctionsDir() {
         writeRoleFile(rolesDir, "data-hub-saved-query-reader.json");
         writeRoleFile(rolesDir, "data-hub-saved-query-writer.json");
         writeRoleFile(rolesDir, "hub-central-downloader.json");
+        writeRoleFile(rolesDir, "hub-central-entity-exporter.json");
```

Comment:
Can we test for this file?
`testInit()` in HubProjectTest already performs such a test.